City-of-Helsinki/drupal-helfi-platform-config
# UHF-2054 Helsinki-kanava oembed

How to test / install:
- Make sure your instance is up and running on latest dev branch.
- Update the modules/themes
   - `composer require drupal/hdbt:dev-UHF-2054_helsinki_kanava && composer require drupal/helfi_platform_config:dev-UHF-2054_helsinki_kanava`
- Run `make drush-updb drush-cr`
- Go to shell `make shell`
  - In shell, type `drush locale:check && drush locale:update && drush cr`
  - Exit shell
- Log in to the site and go to create f.e. Basic page.
   - Add Remote video paragraph
   - Choose a video from helsinki-kanava site. https://www.helsinkikanava.fi/fi_FI/videos
   - Copy the video URL and paste it to the input field and click on Add
   - Once the video has been added to media library, select it from the library and save the page
- View the page and check that the video is rendered

Check also: 
- https://github.com/City-of-Helsinki/drupal-hdbt/pull/225
